src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/utils/PsiTypeUtils.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2019-2020 Red Hat, Inc.
+ * Copyright (c) 2019-2020, 2024 Red Hat, Inc.
```

Comment:
The copyright format for the source files is YEAR1, YEAR2 where YEAR1 is the first year of copyright and YEAR2 is the most recent update. Please fix.

Reply:
@mrglavas updated the header.